qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
 
 .. _issues_unzip:
 
-How to solve unzip errors?
---------------------------
-
-When downloading large zip files within Qiita there is a change that you will get
+How to solve download or unzip errors?
+--------------------------------------
+
+Dealing with large files might be daunting but, in general, following these
+instructions should make things easier. First, make sure that you have enough space
+for the zip download file; for the American Gut, you should have at least close to 900Gb
```

Comment:
This statement will be inaccurate soon. I recommend not listing a size here. 

I did a test against study 2136, and Chrome immediately saw the download size, does the size info just get cached or something?

Reply:
Got it! I thought you meant that we should display beside the button the size, which is the difficult part; if having it in Chrome is sufficient, that's great! Note that the way it works is that python will create the list of files; nginx will take that info, create the zip (and the size reported in Chrome); and Chrome will do the rest. 

Anyway, what about replacing for: if you are unsure of the size required click on the button and your browser will show an estimate size of the download.